src/io/publish/code.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Create output directory
+mkdir -p "$par_output"
+mkdir -p "$par_output_falco"
+
+echo
+echo "par_output:         $par_output"
+echo "par_output_falco:   $par_output_falco"
+echo "par_output_multiqc: $par_output_multiqc"
+
+cp -L "$par_input"/* "$par_output"
+cp -rL "$par_input_falco"/* "$par_output_falco"
+cp -L "$par_input_multiqc" "$par_output_multiqc"
+
```

Comment:
Can we make this abstract by letting the publish component take a list of inputs and a list of outputs, and a loop over them?

Reply:
This does not work because the output should contain a wildcard character.